freqtrade/data/history.py

```diff
@@ -447,18 +442,19 @@ def convert_trades_to_ohlcv(pairs: List[str], timeframes: List[str],
             store_tickerdata_file(datadir, pair, timeframe, data=ohlcv)
 
 
-def get_timeframe(data: Dict[str, DataFrame]) -> Tuple[arrow.Arrow, arrow.Arrow]:
+def get_timerange(data: Dict[str, DataFrame]) -> Tuple[arrow.Arrow, arrow.Arrow]:
```

Comment:
I'll not complain since i think it was me who added this function ... but that was kinda oddly named, ey? :laughing: